### ISSUE TITLE
[doc] Warnings during doc build

### ISSUE THREAD

**Issue by ssato** [ssato/python-anyconfig#104]
Some warnings during doc build need fixes:
```
ssato@x1-carbon-gen6% rm -rf docs/build
ssato@x1-carbon-gen6% make -C docs html
make: ディレクトリ '/home/ssato/repos/public/github.com/ssato/python-anyconfig.git/docs'　に入ります
sphinx-build -b html -d build/doctrees   . build/html
Running Sphinx v1.7.6
   ...
/usr/lib/python2.7/site-packages/anyconfig/backend/base.py:docstring of anyconfig.backend.base.LoaderMixin.allow_primitives:2: WARNING: Field list ends without a blank line; unexpected unindent.
WARNING: autodoc: failed to import module u'anyconfig.backend.json.default'; the following exception was raised:
No module named default
WARNING: autodoc: failed to import module u'anyconfig.backend.json.simplejson'; the following exception was raised:
No module named simplejson
/usr/lib/python2.7/site-packages/anyconfig/backend/toml.py:docstring of anyconfig.backend.toml.Parser._load_from_stream_fn:6: WARNING: Definition list ends without a blank line; unexpected unindent.
   ...
build succeeded, 4 warnings.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
make: ディレクトリ '/home/ssato/repos/public/github.com/ssato/python-anyconfig.git/docs' から出ます
ssato@x1-carbon-gen6%
```

**Comment by ssato** [comment]
update the log:
```
$ LANG=en_US.UTF-8 make -C docs html
make: Entering directory '/home/ssato/repos/public/github.com/ssato/python-anyconfig.git/docs'
sphinx-build -b html -d build/doctrees   . build/html
Running Sphinx v1.7.6
making output directory...
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 37 source files that are out of date
updating environment: 37 added, 0 changed, 0 removed
reading sources... [100%] usage
/usr/lib/python2.7/site-packages/anyconfig/backend/base.py:docstring of anyconfig.backend.base:31: WARNING: Inline interpreted text or phrase reference start-string without end-string.
/usr/lib/python2.7/site-packages/anyconfig/backend/base.py:docstring of anyconfig.backend.base.LoaderMixin.allow_primitives:2: WARNING: Field list ends without a blank line; unexpected unindent.
/usr/lib/python2.7/site-packages/anyconfig/backend/toml.py:docstring of anyconfig.backend.toml.Parser._load_from_stream_fn:6: WARNING: Definition list ends without a blank line; unexpected unindent.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] usage
generating indices... genindex py-modindex
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 3 warnings.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
make: Leaving directory '/home/ssato/repos/public/github.com/ssato/python-anyconfig.git/docs'
$
```

**Comment by ssato** [comment]
I could fix and remove these warnings except for the one which I don't know how to fix by 9b4f898 and so on.

```
$ rm -rf docs/build; make -C docs html
make: ディレクトリ '/home/ssato/repos/public/github.com/ssato/python-anyconfig.git/docs'　に入ります
sphinx-build -b html -d build/doctrees   . build/html
Running Sphinx v1.7.6
making output directory...
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 37 source files that are out of date
updating environment: 37 added, 0 changed, 0 removed
reading sources... [100%] usage
/home/ssato/repos/public/github.com/ssato/python-anyconfig.git/src/anyconfig/backend/toml.py:docstring of anyconfig.backend.toml.Parser._load_from_stream_fn:6: WARNING: Definition list ends without a blank line; unexpected unindent.
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] usage
generating indices... genindex py-modindex
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded, 1 warnings.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
make: ディレクトリ '/home/ssato/repos/public/github.com/ssato/python-anyconfig.git/docs' から出ます
$
```

**Comment by ssato** [comment]
LGTM and closed.
